check_deps/check_cmds_nodes.py

```diff
@@ -30,6 +30,10 @@
                         {
                             "cmd" : tf_cfg.cfg.get("Client", "wrk"),
                             "install" : "wrk"
+                        },
+                        {
+                            "cmd" : tf_cfg.cfg.get("Client", "tls-perf"),
+                            "install" : "tls-perf"
```

Comment:
That's not going to work. This is the package name to install the missing component. But `tls-perf` is domestic utility for TLS handshake performance and cannot be installed from Debian repos.